### PR TITLE
F/wpc 88 update comment

### PR DESCRIPTION
**Pull request by 0815pascal**

### CODE REVIEW

**Review by penk85 — DISMISSED** [review]

**Review by HanaAyman — DISMISSED** [review]

**Review by bishoymelek — APPROVED** [review]

**Review by HanaAyman — APPROVED** [review]

File: packages/editor/src/client/panel/articleMetadataPanel.tsx [hunk 1]
```diff
@@ -67,7 +67,12 @@ export interface ArticleMetadataPanelProps {
   onChange?(value: ArticleMetadata): void
 }
 
-export function ArticleMetadataPanel({value, infoData, onClose, onChange}: ArticleMetadataPanelProps) {
+export function ArticleMetadataPanel({
```
Comment (roosnic1) [inline packages/editor/src/client/panel/articleMetadataPanel.tsx:70]: @bishoymelek i think this file shouldn't be in the PR

File: packages/editor/src/client/routes/articleEditor.tsx [hunk 2]
```diff
@@ -200,8 +200,11 @@ export function ArticleEditor({id}: ArticleEditorProps) {
 
   function countTitle(blocks: BlockValue[]) {
     return blocks.reduce((charLength: number, block: BlockValue) => {
-      if ((block.type === BlockType.Title)) return charLength + (block.value.title as string).length + (block.value.lead as string).length
-      else return charLength;
+      if (block.type === BlockType.Title)
```
Comment (roosnic1) [inline packages/editor/src/client/routes/articleEditor.tsx:203]: @bishoymelek same here.

**Review by roosnic1 — CHANGES_REQUESTED** [review]
Looks all good but there seem to be some files that shouldn't be in the PR.

**Review by HanaAyman — APPROVED** [review]

File: packages/editor/src/client/panel/articleMetadataPanel.tsx [hunk 3]
```diff
@@ -67,12 +67,7 @@ export interface ArticleMetadataPanelProps {
   onChange?(value: ArticleMetadata): void
 }
 
-export function ArticleMetadataPanel({
```
Comment (roosnic1) [inline packages/editor/src/client/panel/articleMetadataPanel.tsx]: @bishoymelek i don't think this file should be changed.

File: packages/editor/src/client/routes/articleEditor.tsx [hunk 4]
```diff
@@ -200,11 +200,8 @@ export function ArticleEditor({id}: ArticleEditorProps) {
 
   function countTitle(blocks: BlockValue[]) {
     return blocks.reduce((charLength: number, block: BlockValue) => {
-      if (block.type === BlockType.Title)
```
Comment (roosnic1) [inline packages/editor/src/client/routes/articleEditor.tsx]: i don't think this file should be changed.

**Review by roosnic1 — CHANGES_REQUESTED** [review]
@bishoymelek this PR still has two files that shouldn't be changed

File: packages/editor/src/client/panel/articleMetadataPanel.tsx (hunk 3 above)
Comment (bishoymelek) [reply packages/editor/src/client/panel/articleMetadataPanel.tsx]: @roosnic1 It has been removed a few days ago, can you double check?

File: packages/editor/src/client/routes/articleEditor.tsx (hunk 4 above)
Comment (bishoymelek) [reply packages/editor/src/client/routes/articleEditor.tsx]: same as above

**Review by roosnic1 — APPROVED** [review]
👍🏾

**Pull request merged**